changelogs/current.yaml

```diff
@@ -2,6 +2,14 @@ date: Pending
 
 behavior_changes:
 # *Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
+- area: eds
+  change: |
+    Introduced caching of EDS assignments when used with ADS. Prior to this change, Envoy required that EDS assignments were sent
+    after an EDS cluster was updated. If no EDS assignment was received for the cluster, it ended up with an empty assignment.
+    Following this change, after a cluster upadate, Envoy waits for an EDS assignment until
```

Comment:
Nit: s/upadate/update/

Reply:
Done. Also updated the comment to reflect that the flag is disabled by default.